pilot/docker/Dockerfile.proxyv2

```diff
@@ -1,9 +1,98 @@
+# If there are not such ARGs in the dockerfile, they will not be passed to the 2nd "FROM" from Makefile.
+#
 # BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
 ARG BASE_DISTRIBUTION=debug
-
 # Version is the base image version from the TLD Makefile
 ARG BASE_VERSION=latest
 
+FROM ubuntu:focal as builder
+
+ARG SDK_VERSION="2.15.100.3"
```

Comment:
I think the latest release is 2.17.100.3.

Reply:
Okay, Fixed